src/main/resources/api.yaml

```diff
@@ -148,6 +143,25 @@ paths:
           description: User do not have permissions to get note details
         '404':
           description: Note not found
+  /notes/{noteId}/board/pages:
+    post:
+      summary: Create a new page in board note
+      operationId: createBoardNotePage
+      tags:
+        - notes
+      parameters:
+        - $ref: '#/components/parameters/noteId'
+      responses:
+          '201':
```

Comment:
thats why i ask about ResponseEntity.ok. We suggest it will be 201 created but return 200 ok